api/src/test/java/org/openmrs/validator/ProviderAttributeTypeValidatorTest.java

```diff
@@ -23,6 +26,20 @@
  */
 public class ProviderAttributeTypeValidatorTest extends BaseContextSensitiveTest {
 	
+	protected static final String PROVIDER_ATTRIBUTE_DATA_XML = "org/openmrs/api/include/ProviderServiceTest-providerAttributes.xml";
```

Comment:
Is there a reason why this is not private?

Reply:
Oversight, thanks. Will edit that